### PR TITLE
Add migration E2E test

### PR DESCRIPTION
**Pull request by danfengliu**
Signed-off-by: danfengl <danfengl@vmware.com>

Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Comment by blackpiglet** [comment]
Github action failure is about go.mod and go.sum files in PR are different from the result from `go mod tidy`, so suggest to run `go mod tidy` command in desktop environment, then patch the change into PR.

File: test/e2e/types.go [hunk 1]
```diff
@@ -47,13 +49,20 @@ type VerleroConfig struct {
...
 	InstallVelero            bool
 	KibishiiDirectory        string
 	Features                 string
 	Debug                    bool
 	GCFrequency              string
+	DefaultCluster           string
+	StandbyCluster           string
+	ClientToBeInstall        TestClient
+	DefaultClient            TestClient
+	StandbyClient            TestClient
```
Comment (qiuming-best) [inline test/e2e/types.go:65]: Maybe it is better to use pointer type (*TestClient)

**Comment by qiuming-best** [comment]
`coverage.out` is it needed to push into our repo?

**Comment by danfengliu** [comment]
> `coverage.out` is it needed to push into our repo?

It's not needed, I will remove it.

File: test/e2e/types.go (hunk 1 above)
Comment (danfengliu) [reply test/e2e/types.go]: Right, good point!

**Review by qiuming-best — APPROVED** [review]

**Review by blackpiglet — APPROVED** [review]

**Pull request merged**